tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -102,8 +102,8 @@ RUN touch ${ROCM_PATH}/.info/version
 COPY install/*.sh /install/
 ARG DEBIAN_FRONTEND=noninteractive
 RUN /install/install_bootstrap_deb_packages.sh
-RUN add-apt-repository -y ppa:openjdk-r/ppa && \
-    add-apt-repository -y ppa:george-edison55/cmake-3.x
+RUN add-apt-repository -y ppa:openjdk-r/ppa
```

Comment:
can we get rid of this line too?

LGTM otherwise

Reply:
Yep, you are right.  That line probably isn't needed anymore.  OpenJDK is installed in the `install_deb_packages.sh` script.